lib/src/Uncertainty/Algorithm/MetaModel/openturns/MetaModelValidation.hxx

```diff
@@ -81,23 +91,20 @@ protected:
 
   void initialize() const;
 
-  /** The input sample */
-  Sample inputSample_;
-
   /** The output sample */
   Sample outputSample_;
 
   /** The meta model */
-  Function metaModel_;
-
-  /** Boolean argument */
-  mutable Bool isInitialized_ = false;
+  Sample metaModelPredictions_;
```

Comment:
do we need to store it, or they can just be evaaluated on the fly wia the metamodel ?

Reply:
Suggestion: 1) Deprecate the old constructor:
```
  MetaModelValidation(const Sample & inputSample,
                      const Sample & outputSample,
                      const Function & metaModel);
```
2) Add it into the ChangeLog ;
3) Add it into the TODO: remove it in the 1.23

Reply:
It is not possible to remove the `metaModelPredictions_` attribute because we cannot compute the metamodel predictions in the case where it is given in the constructor. Since we have decided to deprecate `MetaModelValidation(inputSample, outputSample, metaModel)` because it is less general, the `metaModelPredictions_` attribute must be kept.

Reply:
Fixed at 11eba626ae96dcbdf885e6258ae18039b67e56c4